### ISSUE TITLE
How to change Logo and Description of shimgen exe?

### ISSUE THREAD

**Issue by Dst1502** [chocolatey/shimgen#51]
Similar Problems:  
- https://github.com/chocolatey-archive/chocolatey/issues/579
- https://github.com/chocolatey/shimgen/issues/2
- https://github.com/chocolatey/shimgen/issues/4
My Problem is i don't want the exe generated from shimgen to show their logo and description, but instead use my own logo and desciption.
![attachments](https://user-images.githubusercontent.com/51132809/62524425-e36f2400-b835-11e9-83a4-3fab7db20ec1.png)

**Comment by ferventcoder** [comment]
Hi @Dst1502 - using shimgen.exe within the context of Chocolatey?

**Comment by Dst1502** [comment]
@ferventcoder Yes, ive actually somehow managed to chance the Icon and Firm, but it still says MyName - shim at some places, e.g.: Sloap - shim as Filedescription
